app/uk/gov/hmrc/directdebitupdateemailfrontend/models/audit/EmailSource.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.directdebitupdateemailfrontend.models.audit
+
+import play.api.libs.json.{JsString, Writes}
+
+sealed trait EmailSource {
+
+  val value: String
+
+}
+
+object EmailSource {
+
+  case object New extends EmailSource {
```

Comment:
Could we get away with just using enumeratum here? Like we do on essttp? Although this way doesn't rely on library

Reply:
👍 just updated [here](https://github.com/hmrc/direct-debit-update-email-frontend/pull/14/commits/79687a0079c2fb3a0b3e1d955b6c467dd2e2476c). It does look a lot more concise